src/event.ts

```diff
@@ -50,8 +45,8 @@ export class Event {
    *
    * @returns {Object} Description of the event as an object.
    */
-  asEventDescription(): EventDescription {
-    const description: EventDescription = {
+  asEventDescription(): EventDescription1 {
```

Comment:
Ahhhhh the numbered type has returned!

Reply:
The problem is there are a lot of titles like `Event Name` or `Event Description`.
I could remove the `Event` prefix in those titles to avoid the name clash.

Reply:
Nah, that would be more confusing.